apps/system/style/statusbar/statusbar.css

```diff
@@ -25,9 +25,12 @@
   pointer-events: auto;
 }
 
-#statusbar.hidden {
-  opacity: 0;
-  visibility: hidden;
+
+/* Hide the statusbar out of the viewport in order
+to be available for moz-elements */
+#statusbar.hidden,
+#screen.utility-tray-in-transition > #statusbar {
+  top: calc(var(--statusbar-height) * -1);
```

Comment:
can we cheat with a background-size + background-position to show the translated moz-element?